### DISCUSSION TITLE
Add Kotlin documentation if available

### DISCUSSION THREAD

**Discussion by novaTopFlex** [zealdocs/zeal#1608] [Ideas]
I understand that the "docsets" are provided via Dash for macOS, but I would also like to add documentation that may or may not already be on Dash. If the appropriate documentation for Kotlin is available, could someone please add the documentation to the Zeal browser?

Why is Kotlin such an important programming language? Kotlin, along with other programming languages and accessories developed by JetBrains, is a common programming language for Google Android markets and to a lesser extent on Microsoft Windows markets. Since many programmers are expected to program on a Linux-based system, I would like to have a Kotlin docset option added to Zeal.

**Comment by novaTopFlex** [comment]
Sorry for the interruption, but I have successfully found a workaround that does not require macOS and Dash compilation.

**Reply by Akronix** [reply]
Hi, did you get kotlin in zeal docs? How did you do it?
